www/docs/en/dev/guide/platforms/android/index.md

```diff
@@ -328,13 +328,14 @@ To sign an app, you need the following parameters:
 | Alias                 | `--alias`         | The id specifying the private key used for signing
 | Password              | `--password`      | Password for the private key specified
 | Type of the Keystore  | `--keystoreType`  | *Default: auto-detect based on file extension*<br>Either pkcs12 or jks
+| Package Type          | `--packageType`   | *Default: apk*<br>Builds an APK or a bundle (.aab) file.<br>Accepts either `apk` or `bundle`
```

Comment:
Suggestion by @janpio, with an added word (maybe not needed), spelling fix, and a -> an (spoken language):

```suggestion
| Package Type          | `--packageType`   | *Default: apk*<br>Specify whether to build an APK or an Android App Bundle (`.aab`) file.<br>Accepts either `apk` or `bundle`
```

Reply:
I mentioned it before: Link "Android App Bundle" to https://developer.android.com/guide/app-bundle for context.

Reply:
I have these suggested changes in a pending commit, including the Android App Bundle link